src/main/webapp/i18n/de/calendar.json

```
@@ -0,0 +1,42 @@
+{
+    "artemisApp": {
+        "calendar": {
+            "mondayShort": "Mo",
+            "tuesdayShort": "Di",
+            "wednesdayShort": "Mi",
+            "thursdayShort": "Do",
+            "fridayShort": "Fr",
+            "saturdayShort": "Sa",
+            "sundayShort": "So",
+            "eventSubtypeName": {
+                "lectureStart": "Anfang",
```

```suggestion
                "lectureStart": "Start",
```
consistent with other events always using "Start"